sdks/go/pkg/beam/core/graph/window/trigger.go

```diff
@@ -35,3 +35,38 @@ const (
 	NeverTrigger                           string = "Trigger_Never_"
 	AfterSynchronizedProcessingTimeTrigger string = "Trigger_AfterSynchronizedProcessingTime_"
 )
+
+// TriggerDefault constructs a default trigger that fires after the end of window.
+// No provision for late arriving data.
+func TriggerDefault() Trigger {
+	return Trigger{Kind: DefaultTrigger}
+}
+
+// TriggerAlways constructs an always trigger that keeps firing immediately after an element is processed.
```

Comment:
Consider clarifying that this is equivalent to `TriggerRepeat(TriggerAfterCount(1))`.  Does anything change here WRT late data or not?

Reply:
Okay. Same thing for late data as of default trigger.